performance/collector/cron_job.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+
+DOCKER='/usr/bin/docker'
+DOCKER_CONTAINER='95ea5b320918'
+
+TASK_LOG="/var/log/task.log"
+TASK_FILE="/home/rally/task.yaml"
+
+CONNECTION_STRING='es:///172.18.196.234:9200.myRegion1'
+
+
+TASK_UUID=`${DOCKER} exec ${DOCKER_CONTAINER} \
+rally task start 2>&1 ${TASK_FILE} \
+| tee -a ${TASK_LOG} \
```

Comment:
do we actually need to have file with logs? I mean they are accessibale via docker logs <container_name> and will be stored in elastic search centralized.